src/components/Circles/ThreadPost.tsx

```diff
@@ -20,29 +21,32 @@ export const ThreadPost = ({ post, style }: Props) => {
     theme.spacing.huge;
 
   return (
-    <View style={styles.container}>
-      <View style={styles.header}>
-        <Avatar.Text
-          size={size}
-          style={styles.avatar}
-          label={initials(post?.author?.profile?.displayName)}
-        />
-        <View>
-          <Text style={styles.usernameText}>
-            {post?.author?.profile?.displayName}
-          </Text>
-          {isValid(created) && (
-            <Text style={styles.responseTimeText}>
-              {t('circles.thread-post.responseTime', '{{responseTime}} ago', {
-                responseTime: formatDistanceToNow(created),
-              })}
+    <>
+      {post.priority === Priority.ANNOUNCEMENT && <AnnouncementBanner />}
+      <View style={styles.container}>
+        <View style={styles.header}>
+          <Avatar.Text
+            size={size}
+            style={styles.avatar}
+            label={initials(post?.author?.profile?.displayName)}
+          />
+          <View>
+            <Text style={styles.usernameText}>
```

Comment:
Obviously not to do with this PR, but it will eventually be nice to have these all switched over to Paper components so we get consistency across apps built with this SDK.

Reply:
Yeah for sure. I am trying to be better about that but we will probably need to do a sweep here soon once the circle work is closer to mvp.